### PR TITLE
VIH-9447 ShowOnlyTwentyRecords

### PR DESCRIPTION
**Pull request by abbasiTahir**
as per requirement changed the code to show only 20 records when searching for non-availability hours .

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9447

### Change description ###

added the functionality to show only twenty records when searching for non-availability hours.  

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts [hunk 1]
```diff
@@ -47,14 +47,15 @@ export class VhoSearchComponent implements OnInit {
                         result = await this.service.getNonWorkAvailabilityForVho(this.username.value);
                         break;
                 }
-
-                if (result) {
-                    this.vhoSearchEmitter.emit(result);
+                var myItems = result.slice(0, 20);
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts:50]: Check the sonar cloud analysis results, they have reported the following code smell:

`Unexpected var, use let or const instead`

I'd suggest also using an alternative name and a private property or constant to make the intention clearer. Something like:

`private maxPageSize = 20;`

`const filteredResult = result.slice(0, maxPageSize);`

Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts:50]: We'll need a unit test to check that it emits a maximum of 20 items.

**Review by oliver-scott — COMMENTED** [review]
See comments with some recommended changes. I'd also suggest reverting your changes to `package-lock.json` and `WorkHoursController` as they are not needed

**Review by oliver-scott — APPROVED** [review]

**Review by abbasiTahir — COMMENTED** [review]
ok thank you make sense

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts (hunk 1 above)
Comment (abbasiTahir) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts]: fixed the issues

**Review by abbasiTahir — COMMENTED** [review]
fixed changes in said comment

**Review by abbasiTahir — COMMENTED** [review]
fixed the issue
